src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
-export interface Figure {
+export type FigureColor = 'red' | 'green' | 'blue';
+export type FigureShape = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  shape: FigureShape;
+  color: FigureColor;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: FigureShape = 'triangle';
```

Comment:
and use your enum in each figure
```suggestion
  shape: FigureShape = FigureShape.Triangle;
```